Xamarin.Essentials/Calendars/Calendars.shared.cs

```diff
@@ -47,6 +47,24 @@ public static async Task<CalendarEvent> GetEventAsync(string eventId)
             return await PlatformGetEventAsync(eventId);
         }
 
+        public static Task<CalendarEvent> GetEventInstanceAsync(string eventId, DateTimeOffset instanceDate) => PlatformGetEventInstanceAsync(eventId, instanceDate);
+
+        public static Task<string> CreateCalendarEvent(CalendarEvent newEvent) => PlatformCreateCalendarEvent(newEvent);
+
+        public static Task<bool> UpdateCalendarEvent(CalendarEvent eventToUpdate) => PlatformUpdateCalendarEvent(eventToUpdate);
+
+        public static Task<bool> SetEventRecurrenceEndDate(string eventId, DateTimeOffset recurrenceEndDate) => PlatformSetEventRecurrenceEndDate(eventId, recurrenceEndDate);
```

Comment:
As I look at this `SetEventRecurrenceEndDate`, and the `AddAttendeeToEvent` methods... This feels a bit different to use than .NET.

Maybe we should start thinking about making the calendar object a bit smarter? Or something. I like the idea of Calendar being a POCO, but we are doing a lot of things here that maybe should be on the object instead? I don't know... thoughts are happening in my head.

@Redth @jamesmontemagno this might require better brains than mine.